src/Detector.php

```diff
@@ -64,6 +64,21 @@ public function determineMimeType(\SplFileObject $file): string
         return 'text/plain; charset=utf-8';
     }
 
+    /**
+     * Read width amd height from a image file
```

Comment:
There's a typo here; "Read width **amd** height" (should be "and"?).

Also, could you please end sentences with a period (".").